templates/shop/cart/complete/_apple_pay.html.twig

```diff
@@ -0,0 +1,11 @@
+{% if cw_tpay_get_gateway_config_value(payment.method.gatewayConfig, 'type') == 'apple_pay' %}
+    <input type="hidden" data-apple-pay-amount value="{{ (order.total / 100)|abs }}">
+    <input type="hidden" data-apple-pay-merchant-identifier value="{{ cw_tpay_get_gateway_config_value(payment.method.gatewayConfig, 'apple_pay_merchant_id') }}">
+    <input type="hidden" data-apple-pay-currency value="{{ order.currencyCode }}">
+    <input type="hidden" data-apple-pay-channel-name value="{{ order.channel.name }}" />
```

Comment:
I don't like the 2nd line (operations in Twig) and the 5th one (no way to override it easily).

Reply:
let's put calculation logic into function and lets replace channel name with code?

Reply:
> and lets replace channel name with code?
I use the channel name to display it while paying with Apple Pay. Channel code won't fit here.

Reply:
I've created the `cw_tpay_convert_minor_to_major_currency` filter and now it's look like this:
```twig
<input type="hidden" data-apple-pay-amount value="{{ order.total|cw_tpay_convert_minor_to_major_currency }}">
```